js/contribution.js

```diff
@@ -157,8 +149,7 @@ class JS extends Contribution {
     this.fuzzPublishers()
     this.state = ledgerState.setInfoProp(this.state, 'reconcileStamp', 1528948800000)
     this.ledger.run(this.state, 0)
-    this.staticizeBallots()
-    return ledgerState.getInfoProp(this.state, 'transactions') || []
+    return this.state.getIn(['ledger', 'info'])
```

Comment:
There is this:

https://github.com/brave-intl/bat-ledger-spec/blob/master/js/spec.js#L102

Inside spec.js, which this class inherits, could we maybe use this/refactor current method perhaps?

Reply:
Yes. Can use it as-is. Done.